src/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -586,47 +575,35 @@ public float GetSaturation()
             return s;
         }
 
-        public int ToArgb()
-        {
-            return unchecked((int)Value);
-        }
+        public int ToArgb() => unchecked((int)Value);
 
-        public KnownColor ToKnownColor()
-        {
-            return (KnownColor)knownColor;
-        }
+        public KnownColor ToKnownColor() => (KnownColor)knownColor;
 
         public override string ToString()
         {
             if ((state & s_stateNameValid) != 0 || (state & s_stateKnownColorValid) != 0)
             {
-                return nameof(Color) + " [" + Name + "]";
+                return $"{nameof(Color)} [{Name}]";
             }
             else if ((state & s_stateValueMask) != 0)
             {
-                return nameof(Color) + " [A=" + A.ToString() + ", R=" + R.ToString() + ", G=" + G.ToString() + ", B=" + B.ToString() + "]";
+                return $"{nameof(Color)} [A={A}, R={R}, G={G}, B={B}]";
             }
             else
             {
-                return nameof(Color) + " [Empty]";
+                return $"{nameof(Color)} [Empty]";
```

Comment:
Previously this was returning a string constant, now it's going through `string.Format` and allocating a new string each time.